runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -628,6 +669,24 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
             }
 #endif // defined(J9VM_INTERP_AOT_COMPILE_SUPPORT) && defined(J9VM_OPT_SHARED_CLASSES) && (defined(TR_HOST_X86) || defined(TR_HOST_POWER) || defined(TR_HOST_S390) || defined(TR_HOST_ARM) || defined(TR_HOST_ARM64))
          } // if (TR::Options::sharedClassCache())
+#if defined(J9VM_OPT_JITSERVER)
+      else if (methodCachedAtServer) // Not cached in SCC but the server
```

Comment:
We would like to set a low count either when SCC/AOT is not set, or when it is set but the method is not in local SCC.
We may have to use the code below in more than one place. Maybe define it in a function?